src/jmcomic/jm_entity.py

```
@@ -164,6 +164,32 @@ def get_dirname(cls, detail: 'DetailEntity', ref: str) -> str:
+            result[prefix + k] = v
+
+        # property
+        for cls in inspect.getmro(type(self)):
+            for name, attr in cls.__dict__.items():
+                k = prefix + name
+                if k not in result and isinstance(attr, property):
+                    v = attr.__get__(self, cls)
+                    result[k] = v
+
+        # advice
+        advice_dict = JmModuleConfig.AFIELD_ADVICE if self.is_album() else JmModuleConfig.PFIELD_ADVICE
```

Ruff (0.8.2)

`JmModuleConfig` may be undefined, or defined from star imports

(F405)